hw2/03-count.html

```diff
@@ -9,16 +9,16 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="03-count.js"></script>
   </head>
-  <body>
-    <main>
-      <h1>Exercise 03 - Count</h1>
+  <body class="bg-primary container mt-5">
```

Comment:
Instead of using the `bg-primary` for the background color I would use `style= "background-color: #01435C;"` for matching the color in the example pictures.